pkg/controller/rpaasinstance/rpaasinstance_controller.go

```diff
@@ -41,9 +42,41 @@ import (
 )
 
 const (
-	defaultConfigHistoryLimit = 10
-
+	defaultConfigHistoryLimit     = 10
+	defaultCacheHeaterCronImage   = "bitnami/kubectl:latest"
+	defaultCacheHeaterSchedule    = "* * * * *"
 	defaultPortAllocationResource = "default"
+	volumeTeamLabel               = "tsuru.io/volume-name"
+
+	cacheHeaterCronJobSuffix = "-heater-cron-job"
+	cacheHeaterVolumeSuffix  = "-heater-volume"
+
+	rsyncCommandPodToPVC = "rsync -avz --recursive --delete --temp-dir=/var/cache/cache-heater/temp /var/cache/nginx/rpaas/nginx /var/cache/cache-heater"
```

Comment:
The path for the cache directory should be based on the `plan.Spec.Config.CachePath` field.

Reply:
done!